inference-engine/tests/functional/plugin/myriad/ngraph/operations/static_shape_broadcast.cpp

```diff
@@ -137,6 +166,31 @@ INSTANTIATE_TEST_CASE_P(smoke_NGraph, StaticShapeBroadcastExplicitTests, testing
         testing::ValuesIn(testExplicitStaticShapes))
 );
 
+
+TEST_P(StaticShapeBroadcastBidirectionalTests, CanBeConstrocted) {
+    const auto shapeOf = std::make_shared<ngraph::opset3::ShapeOf>(m_tensorWithTargetShape);
+    std::shared_ptr<ngraph::vpu::op::StaticShapeBroadcast> op;
+    ASSERT_NO_THROW(op = std::make_shared<ngraph::vpu::op::StaticShapeBroadcast>(
+            m_tensor, shapeOf, ngraph::op::BroadcastType::BIDIRECTIONAL));
+}
+
+TEST_P(StaticShapeBroadcastBidirectionalTests, CanValidateAndInferTypes) {
+    const auto shapeOf = std::make_shared<ngraph::opset3::ShapeOf>(m_tensorWithTargetShape);
+    std::shared_ptr<ngraph::vpu::op::StaticShapeBroadcast> op;
+    ASSERT_NO_THROW(op = std::make_shared<ngraph::vpu::op::StaticShapeBroadcast>(
+            m_tensor, shapeOf, ngraph::op::BroadcastType::BIDIRECTIONAL));
+    ASSERT_NO_THROW(std::make_shared<ngraph::Function>(
+            ngraph::OutputVector{op->output(0)},
+            ngraph::ParameterVector{m_tensor, m_tensorWithTargetShape}));
+    ASSERT_EQ(m_tensorWithTargetShape->get_shape(), op->output(0).get_shape());
```

Comment:
It's strange for me that it passes, because the target shape and the result shape are generally different in `BIDIRECTIONAL` case.

Reply:
I've removed this check